openedx/core/djangoapps/credentials/models.py

```diff
@@ -17,35 +19,36 @@ class CredentialsApiConfig(ConfigurationModel):
     Manages configuration for connecting to the Credential service and using its
     API.
     """
+
     class Meta(object):
-        app_label = "credentials"
+        app_label = 'credentials'
 
     OAUTH2_CLIENT_NAME = 'credentials'
     API_NAME = 'credentials'
     CACHE_KEY = 'credentials.api.data'
 
-    internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
-    public_service_url = models.URLField(verbose_name=_("Public Service URL"))
+    internal_service_url = models.URLField(verbose_name=_('Internal Service URL'))
+    public_service_url = models.URLField(verbose_name=_('Public Service URL'))
```

Comment:
Looks like these fields are deprecated now. Can you try to keep this consistent with however you choose to indicate that deprecation in https://github.com/edx/edx-platform/pull/15180? Updating the help text might be a good way to go.